### PR TITLE
Cleaned up some code

### PR DESCRIPTION
**Pull request by ElianCordoba**
- Removed unnecessary
- Changed any type for actual types

### CODE REVIEW

**Comment by santiq** [comment]
LGTM

File: src/jobs/emailSequence.ts [hunk 1]
```diff
@@ -3,12 +3,12 @@ import MailerService from '../services/mailer';
...
 
-  public async handler(job, done): Promise<any> {
+  public async handler(job, done): Promise<void> {
     console.log('✌️ Email Sequence Job triggered!');
     try {
-      const { email, name } = job.data;
+      const { email, name }: { [key: string]: string } = job.data;
       const mailerServiceInstance = Container.get(MailerService);
-      await mailerServiceInstance.StartEmailSequence(email, name);
+      await mailerServiceInstance.StartEmailSequence(email, name as any); // Here you are passing name but in the function you spect user
```
Comment (ElianCordoba) [inline src/jobs/emailSequence.ts]: What do you think of this?

File: src/api/routes/auth.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ export default (app) => {
 
   route.post('/signin', async (req: Request, res: Response, next: NextFunction) => {
     try {
-      const { email, password } = req.body;
+      const { email, password }: { [key: string]: string } = req.body;
```
Comment (santiq) [inline src/api/routes/auth.ts]: @ElianCordoba remove this typing

File: src/jobs/emailSequence.ts (hunk 1 above)
Comment (santiq) [reply src/jobs/emailSequence.ts]: That is a bug, it should be: 
`mailerServiceInstance.StartEmailSequence('WelcomeSequence', { email, name})`

Comment (santiq) [reply src/jobs/emailSequence.ts]: Would you mind in making the fix?

Comment (ElianCordoba) [reply src/jobs/emailSequence.ts]: Sure

**Comment by ElianCordoba** [comment]
@santiq  Done

**Pull request merged**